frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersionListView.tsx

```diff
@@ -81,95 +100,122 @@ const ModelVersionListView: React.FC<ModelVersionListViewProps> = ({
-      toolbarContent={
-        <ToolbarContent>
-          <ToolbarToggleGroup toggleIcon={<FilterIcon />} breakpoint="xl">
-            <ToolbarGroup variant="filter-group">
-              <ToolbarFilter
-                chips={search === '' ? [] : [search]}
-                deleteChip={() => setSearch('')}
-                deleteChipGroup={() => setSearch('')}
-                categoryName={searchType}
-              >
-                <SimpleSelect
-                  dataTestId="model-versions-table-filter"
-                  options={searchTypes.map((key) => ({
-                    key,
-                    label: key,
-                  }))}
-                  value={searchType}
-                  onChange={(newSearchType) => {
-                    const enumMember = asEnumMember(newSearchType, SearchType);
-                    if (enumMember !== null) {
-                      setSearchType(enumMember);
-                    }
-                  }}
-                  icon={<FilterIcon />}
-                />
-              </ToolbarFilter>
-              <ToolbarItem variant="search-filter">
-                <SearchInput
-                  placeholder={`Find by ${searchType.toLowerCase()}`}
-                  value={search}
-                  onChange={(_, searchValue) => {
-                    setSearch(searchValue);
-                  }}
-                  onClear={() => setSearch('')}
-                  style={{ minWidth: '200px' }}
-                  data-testid="model-versions-table-search"
-                />
-              </ToolbarItem>
-            </ToolbarGroup>
-          </ToolbarToggleGroup>
-          <ToolbarItem>
-            <Button
-              variant="secondary"
-              onClick={() => {
-                navigate(registerVersionForModelUrl(rm?.id, preferredModelRegistry?.metadata.name));
-              }}
-            >
-              Register new version
-            </Button>
-          </ToolbarItem>
-          <ToolbarItem>
-            <Dropdown
-              isOpen={isArchivedModelVersionKebabOpen}
-              onSelect={() => setIsArchivedModelVersionKebabOpen(false)}
-              onOpenChange={(isOpen: boolean) => setIsArchivedModelVersionKebabOpen(isOpen)}
-              toggle={(tr: React.Ref<MenuToggleElement>) => (
-                <MenuToggle
-                  data-testid="model-versions-table-kebab-action"
-                  ref={tr}
-                  variant="plain"
-                  onClick={() =>
-                    setIsArchivedModelVersionKebabOpen(!isArchivedModelVersionKebabOpen)
-                  }
-                  isExpanded={isArchivedModelVersionKebabOpen}
-                  aria-label="View archived versions"
+    <>
+      {isArchiveModel && (
+        <Alert
+          variant="warning"
+          isInline
+          title={`All the versions have been archived along with the model by ${
+            rm?.owner ?? '--'
+          } on  ${
```

Comment:
@yih-wang @vconzola I don't think we can actually display who the models/versions were archived by. This code will display that they have been archived by whoever is the model owner, which may not be true if multiple users have access to the registry and someone else archived it. The API currently has no way to record who made a change unless we change the owner when someone archives something, which doesn't seem right.

I think maybe we need to instead not name a user here and say "All the versions have been archived along with the model on [date]". What do you think?

In the future we could ask the MR team to add a `lastModifiedBy` field to the API for us to track who made changes.

Reply:
That makes sense to me since we don't record any model activities for now. 

> I think maybe we need to instead not name a user here and say "All the versions have been archived along with the model on [date]". What do you think?
Sounds good!